instrumentation/mongo/mongo-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/mongo/MongoClientTracer.java

```diff
@@ -84,6 +84,29 @@ protected InetSocketAddress peerAddress(CommandStartedEvent event) {
     }
   }
 
+  @Override
+  protected String spanName(
+      CommandStartedEvent event, BsonDocument document, String normalizedQuery) {
+    String dbName = dbName(event);
+    if (event.getCommandName() == null) {
+      return dbName == null ? DB_QUERY : dbName;
+    }
+
+    String collectionName = collectionName(event);
+    StringBuilder name = new StringBuilder();
+    name.append(event.getCommandName()).append(' ');
```

Comment:
Does this have a chance of a span name with a trailing space? Perhaps we can add a unit test for that case.

Reply:
Ah, good call. There are operations that can probably be done through the Java driver that wouldn't have a DB Name. I'll update and add a test for that. Thanks!

Reply:
I've updated this and added a test that returns null for both dbName and the collection name. I don't know if its possible to actually have a situation with no dbName (even management commands use `admin` as the dbName) but its good to be safe :)